CHANGELOG.md

```diff
@@ -5,22 +5,41 @@ All notable changes to this project will be documented in this file.
+## [0.7.0] -  2024-06-20
 
-### Fixed
-
-- Fix a deprecation warning in PySide6, thanks to [@naglis](https://github.com/naglis) ([issue #595](https://github.com/freedomofpress/dangerzone/issues/595))
-- Make update notifications work in systems with PySide2, thanks to [@naglis](https://github.com/naglis) ([issue #788](https://github.com/freedomofpress/dangerzone/issues/788))
-
-### Security
+### Added
 
 - Integrate Dangerzone with gVisor, a memory-safe application kernel, thanks to [@EtiennePerot](https://github.com/EtiennePerot) ([#126](https://github.com/freedomofpress/dangerzone/issues/126))
-  As a result of this integration, we have also improved Dangerzone's security
-  in the following ways:
+  As a result of this integration, we have also improved Dangerzone's security in the following ways:
   * Prevent attacker from becoming root within the container ([#224](https://github.com/freedomofpress/dangerzone/issues/224))
   * Use a restricted seccomp profile ([#225](https://github.com/freedomofpress/dangerzone/issues/225))
   * Make use of user namespaces ([#228](https://github.com/freedomofpress/dangerzone/issues/228))
+- Added some articles about the Dangerzone project to the `README.md` ([#807](https://github.com/freedomofpress/dangerzone/pull/807))
+- Explained how to create, sign, and verify source tarballs ([#823](https://github.com/freedomofpress/dangerzone/pull/823))
+- Added a design doc for the update notifications
+- Added a design doc for the gVisor integration ([#815](https://github.com/freedomofpress/dangerzone/pull/815))
+
+### Fixed
+
+- Fix a deprecation warning in PySide6, thanks to [@naglis](https://github.com/naglis) ([issue #595](https://github.com/freedomofpress/dangerzone/issues/595))
+- Make update notifications work in systems with PySide2, thanks to [@naglis](https://github.com/naglis) ([issue #788](https://github.com/freedomofpress/dangerzone/issues/788))
+- Bumped Python system path to 3.12 in Dockerfile ([#812](https://github.com/freedomofpress/dangerzone/pull/812))
+- Detect bad file permissions in RPMs ([issue #727](https://github.com/freedomofpress/dangerzone/pull/727))
+- Quote commands in installation instructions, making it compatible with `zsh` based shells. (issue [#805](https://github.com/freedomofpress/dangerzone/issues/805))
+- Removed the python shebang from some files
+ - Changed how tests fixtures are used, fixing an uncovered bug along the way ([issue #813](https://github.com/freedomofpress/dangerzone/pull/813))
+- Order the list of PDF viewers and return the default application first on Linux, thanks to [@rocodes](https://github.com/rocodes) (issue [#814](https://github.com/freedomofpress/dangerzone/pull/814))
+- Use the `LF` line ending for all content except images ([#838](https://github.com/freedomofpress/dangerzone/pull/838))
+- Bumped the minimum python version to 3.9, due to Pyside6 dropping support for python 3.8 ([#780](https://github.com/freedomofpress/dangerzone/pull/780))
```

Comment:
```suggestion
- Development: Bumped the minimum python version to 3.9, due to Pyside6 dropping support for python 3.8 ([#780](https://github.com/freedomofpress/dangerzone/pull/780))
```

Reply:
On this one, I'm not sure this is related to "Development", as it seems more related to packaging :thinking:. I would actually let it without the "Development" prefix here.

Reply:
Actually, makes sense wrt your other comment

Reply:
Huh, you're righ _scratches head_. It's fine that way, not sure why I added the "Development:" tag there.